stl/inc/format

```diff
@@ -1001,15 +1001,15 @@ inline constexpr char32_t _Width_estimate_intervals[] = { //
 
 _NODISCARD constexpr int _Unicode_width_estimate(const char32_t _Ch) noexcept {
     // Computes the width estimation for Unicode characters from N4950 [format.string.std]/13
-    if (_Ch < 0x25FDu) {
+    constexpr char32_t _Limit = _Width_estimate_intervals[12];
+    if (_Ch < _Limit) {
         int _Result = 1;
         for (const auto& _Bound : _Width_estimate_intervals) {
             if (_Ch < _Bound) {
                 return _Result;
             }
             _Result ^= 0b11u; // Flip between 1 and 2 on each iteration
         }
-

```

Comment:
I think a comment to clarify that the `else` branch is just an optimization will be helpful.